recipes-tag/redis-py/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "redis-py" %}
```

Comment:
We already have this on the anaconda channel.

Reply:
not sure how i missed it (might have forgotten `-c anaconda` when manually building for lightflow the first time around). just pushed the changes.
I also searched for all these packages on the anaconda channel : `anaconda.org/anaconda` and did not find others. good catch!